lib/index.js

```diff
@@ -0,0 +1,40 @@
+import setupHtmlToArticleJson from 'html-to-article-json';
+import articleJsonToAmp from 'article-json-to-amp';
+import Promise from 'pinkie-promise';
```

Comment:
@iefserge pinkie-promise return the native promise if available. I saw someone using it so I figured it'd make sense. I'm fine using Bluebird if you have experience with that and you know it's good!